public/templates/templates.json

```diff
@@ -189,15 +189,15 @@
       "author": "tr1ten",
       "language": "Java",
       "published": "2021-01-01",
-      "src": "java/IndexPriorityQueue.java"
+      "src": "java/index_priority_queue.java"
```

Comment:
Sorry for inconvenience, but i think it would be better to keep java file name as their classname to avoid any confusion. 
So please avoid renaming Java one's to small as well.

Reply:
We are not running this code, so i dont think it matters if we change the case or not. If all files follow the same case, it looks consistent.